fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/ShowColumnStatsCommand.java

```diff
@@ -0,0 +1,422 @@
+import org.apache.doris.nereids.trees.plans.commands.info.TableNameInfo;
+import org.apache.doris.nereids.trees.plans.visitor.PlanVisitor;
+import org.apache.doris.qe.ConnectContext;
+import org.apache.doris.qe.ShowResultSet;
+import org.apache.doris.qe.ShowResultSetMetaData;
+import org.apache.doris.qe.StmtExecutor;
+import org.apache.doris.statistics.AnalysisManager;
+import org.apache.doris.statistics.ColStatsMeta;
+import org.apache.doris.statistics.ColumnStatistic;
+import org.apache.doris.statistics.PartitionColumnStatistic;
+import org.apache.doris.statistics.PartitionColumnStatisticCacheKey;
+import org.apache.doris.statistics.ResultRow;
+import org.apache.doris.statistics.StatisticsRepository;
+import org.apache.doris.statistics.TableStatsMeta;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.Lists;
+import com.google.common.collect.Sets;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+/**
+ * Represents the command for SHOW COLUMN STATS.
+ */
+public class ShowColumnStatsCommand extends ShowCommand {
+    private static final Logger LOG = LogManager.getLogger(ShowColumnStatsCommand.class);
+    private static final ImmutableList<String> TABLE_COLUMN_TITLE_NAMES =
+            new ImmutableList.Builder<String>()
+                    .add("column_name")
+                    .add("index_name")
+                    .add("count")
+                    .add("ndv")
+                    .add("num_null")
+                    .add("data_size")
+                    .add("avg_size_byte")
+                    .add("min")
+                    .add("max")
+                    .add("method")
+                    .add("type")
+                    .add("trigger")
+                    .add("query_times")
+                    .add("updated_time")
+                    .add("update_rows")
+                    .add("last_analyze_row_count")
+                    .add("last_analyze_version")
+                    .build();
+
+    private static final ImmutableList<String> PARTITION_COLUMN_TITLE_NAMES =
+            new ImmutableList.Builder<String>()
+                    .add("column_name")
+                    .add("partition_name")
+                    .add("index_name")
+                    .add("count")
+                    .add("ndv")
+                    .add("num_null")
+                    .add("min")
+                    .add("max")
+                    .add("data_size")
+                    .add("updated_time")
+                    .add("update_rows")
+                    .add("trigger")
+                    .build();
+
+    private final TableNameInfo tableName;
+    private final List<String> columnNames;
+    private final PartitionNames partitionNames;
```

Comment:
```suggestion
    private final PartitionNamesInfo partitionNames;
```
use PartitionNamesInfo instead

Reply:
done - pls review